.gitignore

```diff
@@ -1 +1,2 @@
 *.sw[o-p]
+vendor
```

Comment:
We don't want to ignore the vendor directory. Eventually (in another PR) all the vendored packages should be updated. I think this will require some code updates too, but shouldn't be too bad.